internal/nostr/nostr.go

```diff
@@ -203,6 +204,67 @@ func (svc *Service) InfoHandler(c echo.Context) error {
 	}
 }
 
+func (svc *Service) PublishHandler(c echo.Context) error {
+	var requestData PublishRequest
+	if err := c.Bind(&requestData); err != nil {
+		return c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: fmt.Sprintf("error decoding publish request: %s", err.Error()),
+		})
+	}
+
+	if len(requestData.Relays) == 0 {
+		return c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: "relay(s) not specified",
+		})
+	}
+
+	if (requestData.SignedEvent == nil) {
+		return c.JSON(http.StatusBadRequest, ErrorResponse{
+			Message: "signed event is empty",
+		})
+	}
+
+	var wg sync.WaitGroup
+	errorCh := make(chan error, len(requestData.Relays))
+
+	for _, relayUrl := range requestData.Relays {
```

Comment:
let's support here only one relay (as in all other endpoints I think) and if no relay is given we use the default realy.

Reply:
Changed to support only one relay now!